pkgs/development/python-modules/qtile/default.nix

```diff
@@ -70,6 +71,7 @@ buildPythonPackage rec {
     dbus-next
     dbus-python
```

Comment:
I think dbus-python is unused and can be removed.

Reply:
One of the qile maintainers here, correct. Good catch